client/src/store/posts.js

```diff
@@ -6,25 +6,25 @@ const state = {
     id_1: {
       username: 'socleansofreshh',
       userImage: 'https://s3-us-west-2.amazonaws.com/s.cdpn.io/1211695/me_3.jpg',
-      postVideo: 'https://firebasestorage.googleapis.com/v0/b/emocean-74133.appspot.com/o/profiles%2Fvideoplayback.mp4?alt=media&token=1595d9f7-9e0c-459b-ae25-06ee4ac19cb2',
-      likes: 36,
+      postVideo: '',
+      likes: 3632,
```

Comment:
Why you still using hardcoded values? You already have db. If you don't have them in db harcode it there.